### PR TITLE
Add minisite_url metadata for Arapey, Fraunces, Gulzar, Recursive, Space Grotesk

### PR DESCRIPTION
**Pull request by chrissimpkins**
Initial batch of minisite_url metadata updates.

### CODE REVIEW

File: ofl/arapey/METADATA.pb [hunk 1]
```diff
@@ -24,3 +24,4 @@ fonts {
 subsets: "menu"
 subsets: "latin"
 subsets: "latin-ext"
+minisite_url: "https://arapey.xyz"
```
Comment (davelab6) [inline ofl/arapey/METADATA.pb:27]: This site is for the new opsz upgrade which hasn't started to be onboarded, so I suggest to remove this here and keep it on an internal tracker of upcoming minisites.

**Review by davelab6 — APPROVED** [review]
I suggest reverting Arapey and then the other 4 LGTM

File: ofl/gulzar/METADATA.pb [hunk 2]
```diff
@@ -21,3 +21,4 @@ source {
   repository_url: "https://github.com/simoncozens/Gulzar"
   commit: "7f736710573840523e68ea977843f7af7239632b"
 }
+minisite_url: "https://gulzarfont.org"
```
Comment (RosaWagner) [inline ofl/gulzar/METADATA.pb:24]: There is an open PR for Gulzar #5530, we could avoid conflict with adding the minisite there

**Comment by applecuckoo** [comment]
Hello there! I'm not sure if this is the right place to say this, but Nabla has https://nabla.typearture.com/ which you could also add in this PR.

**Comment by chrissimpkins** [comment]
Thank you!

File: ofl/recursive/METADATA.pb [hunk 3]
```diff
@@ -46,3 +46,4 @@ source {
   repository_url: "https://github.com/arrowtype/recursive"
   commit: "071fc21f217781110d67e8d0bf5021f31cbdcb85"
 }
+minisite_url: "https://www.recursive.design"
```
Comment (davelab6) [inline ofl/recursive/METADATA.pb:49]: No www on this

File: ofl/gulzar/METADATA.pb (hunk 2 above)
Comment (RosaWagner) [reply ofl/gulzar/METADATA.pb:24]: Since Gulzar is blocked for now, let's add the mini site here and deal with the conflict later. cc @simoncozens

**Comment by chrissimpkins** [comment]
> Hello there! I'm not sure if this is the right place to say this, but Nabla has https://nabla.typearture.com/ which you could also add in this PR.

@applecuckoo We'll add this one in a subsequent PR.  Thanks again!

**Comment by applecuckoo** [comment]
@chrissimpkins hasn't happened yet, should I make an issue just to keep track?

**Pull request merged**